src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/_params.py

```diff
@@ -0,0 +1,72 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from argcomplete.completers import FilesCompleter
+from azure.cli.core.commands import \
+    (register_cli_argument, CliArgumentType)
+from azure.cli.core.commands.parameters import \
+    (tags_type,
+     get_resource_name_completion_list,
+     resource_group_name_type,
+     enum_choice_list)
+
+from azure.cli.command_modules.dla._validators import (validate_resource_group_name,
```

Comment:
@derekbekoe and @troydai I am following a validator pattern for iso date time strings that is also in some other validators, so this should be ok (see the batch validators). Also, here are the current load times for DLA and DLS:
Loaded module 'dla' in 0.418 seconds.
Loaded module 'dls' in 0.480 seconds.

vs batch: Loaded module 'batch' in 1.096 seconds.

A lot of modules load much faster, but it looks like we are right around the middle of the pack now.